<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Таблица:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Таблица</string>
    </property>
    <addaction name="newTabAc"/>
    <addaction name="delTabAc"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Запись</string>
    </property>
    <addaction name="addWriteAc"/>
    <addaction name="delWriteAc"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <action name="openAc">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="newTabAc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Создать новую</string>
   </property>
  </action>
  <action name="delTabAc">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="addWriteAc">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="delWriteAc">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
